Add PDF generation for datasheets

anatoly-particle opened the pull request.

anatoly-particle committed:
Add PDF generation for datasheets

monkbroc commented on scripts/pdf-generation/README.md:
```diff
@@ -0,0 +1,107 @@
+
+**But make sure that you've done following things before:**
+- Install all the **dependencies** with `npm install`
+- Install **wkhtmltopdf** tool for your system from https://wkhtmltopdf.org/downloads.html
```
What version do you use? We should document that.

monkbroc committed:
Remove Babel and use native ES6 syntax for Node 6+

monkbroc committed:
Update scripts for PDF generation

monkbroc reviewed (approved):
:+1:

anatoly-particle committed:
Change font, add PDFs, fix SVG, fix relative links to website
— fixed Electron's cover SVG
— set Montserrat as main font (that looks almost the same as Gotham, but is free and fixed width issue)
— relative links not to images now point to docs.particle.io/… resources
— style changes and fixes: page breaks before headings, links around images, unused blockquotes, TOC styling
— small stuff: detached cover styles into a separate stylesheet, added pre-generated PDFs, readme fixes

monkbroc committed:
Finish Photon datasheet formatting

monkbroc committed:
Page breaks for P1 and Electron

anatoly-particle committed:
Update PDFs
